_guides/responsive/control_layout.md

```diff
@@ -137,7 +121,7 @@ Depending on the screen width, one or the other will be fetched and rendered.
 </amp-img>
 {% endhighlight %}
 
-## Control element’s size and assets using srcset and sizes
+## Using srcset and sizes
```

Comment:
I'm not really sure about mentioning Safari here? Should it be a warning of some type? That's how I'm treating it, but it feels a little weird.